.github/workflows/integration-check-install.yml

```diff
@@ -5,17 +5,26 @@ on:
 
 jobs:
   install:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
Should we have same checks for windows?

Reply:
Planned to add them , windows with gitbash for now.